throttle/contract_tx_limiter_middleware.go

```diff
@@ -179,12 +237,19 @@ func NewContractTxLimiterMiddleware(cfg *ContractTxLimiterConfig,
 			}
 			txl.tierMap[contractTierID] = tierInfo
 		}
+		tierMapLoadDuration := time.Since(start)
+		lvs := []string{"method", "tierMap Load", "error", fmt.Sprint(err != nil)}
+		tierMapLoadLatency.With(lvs...).Observe(tierMapLoadDuration.Seconds())
```

Comment:
Don't think there's any point in having this metric either, it only loads one tier each time.

Reply:
Get Tier Map does a range query with tier key prefix and loads all the tierId --> TierInfo binding.
Later part of the code fetches a specific TierId data, so I think we should exclude this part and keep loading time measured from above

Reply:
Instrument the code where `GetTierMap` is called, the `GetTierInfo` latency can be ignored.

Reply:
Done